vere/sist.c

```diff
@@ -1262,6 +1275,10 @@ u3_sist_boot(void)
 
     u3A->own = who;
 
+    //  set single-home
```

Comment:
We don't set single-home for fake ships?

Reply:
We do, but that function is kind of a mess. The last few lines from (`u3A->own = who;` to the end) run for every kind of first-boot.